cobigen-templates/templates-devon4j/src/main/templates/crud_angular_client_app/templates.xml

```diff
@@ -27,7 +26,6 @@
       <incrementRef ref="app_angular_devon4ng_component"/>
     </increment>
     <increment name="app_angular_env" description="Angular devon4ng URL">
-        <templateRef ref="environment.ts"/>
         <templateRef ref="environment.prod.ts"/>
```

Comment:
You will also have to remove environment.prod.ts file as it is closely related

Reply:
did not change it, since there is a variable in it.

  restPathRoot: 'http://localhost:8081/**${variables.domain}**-server/',
  restServiceRoot: 'http://localhost:8081/**${variables.domain}**-server/services/rest/'

if this is not an issue, prod file can be removed.

Reply:
Yes, it can be removed. The whole increment should be removed as there are no templates to generate.